libs/coin-modules/coin-evm/src/__tests__/coin-tester/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.8'
```

Comment:
To be checked if we really want to upgrade to v3 as, as I mentionned on Stephane's comment, [v2 is still the recommended version for development without docker swarm](https://forums.docker.com/t/limit-resources-without-swarm/98524/2).